test/shifts_test.rb

```diff
@@ -11,21 +11,27 @@ def test_it_exists
   end
 
   def test_it_can_generate_a_random_number
+    assert_equal true, @shift.random_number.to_s.length <= 5
```

Comment:
These "around" methods are especially important in terms of achieving 100% simplecov coverage!